src/FsAutoComplete/FsAutoComplete.fsproj

```diff
@@ -15,10 +15,18 @@
     <PackageProjectUrl>$(RepositoryUrl)</PackageProjectUrl>
     <Authors>FsAutoComplete contributors</Authors>
     <SatelliteResourceLanguages>en</SatelliteResourceLanguages>
+    <IsPackable>true</IsPackable>
+    <PackAsTool>true</PackAsTool>
```

Comment:
combined, this means that when this project is packaged, it's done as a tool. before, we did this in the FAKE script for some odd reason.